docs/How-To/cli/README.md

```diff
@@ -1,41 +1,45 @@
 ---
 Title: Budgetd
-Description: A high-level overview of how the command-line (CLI) interfaces work for the budget module.
+Description: Description of governance proposal for a budget module command line interface (CLI).
 ---
 
 # Budgetd
 
-This document provides a high-level overview of how the command-line (CLI) interfaces work for the budget module.
+The budgetd binary does not include a CLI. This document describes a governance proposal for a budget module command line interface (CLI).
```

Comment:
To be precise, the `budgetd` binary only includes query commands, not transaction commands. Users can query the values set as budget parameters, query budget plans, and query an address that can be used as source or destination address.

https://github.com/tendermint/budget/blob/def71fde8cc829f18de2db21ae695d0a8109620a/x/budget/client/cli/query.go#L18-L35